index.js

```diff
@@ -11,68 +11,145 @@ module.exports = {
+    'import/no-unresolved': 'off',
 
     // Allow paren-less arrow functions only when there's no braces
-    'arrow-parens': [2, 'as-needed', { requireForBlockBody: true }],
+    'arrow-parens': ['error', 'as-needed', { requireForBlockBody: true }],
 
     // Allow async-await
-    'generator-star-spacing': 0,
+    'generator-star-spacing': 'off',
 
     // Allow debugger during development
-    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
-    'no-console': process.env.NODE_ENV === 'production' ? 2 : 0,
+    'no-debugger': process.env.NODE_ENV === 'production' ? 'error' : 'off',
+    'no-console': process.env.NODE_ENV === 'production' ? 'error' : 'off',
 
     // Prefer const over let
-    'prefer-const': [2, {
+    'prefer-const': ['error', {
       'destructuring': 'any',
       'ignoreReadBeforeAssign': false
     }],
 
     // No single if in an "else" block
-    'no-lonely-if': 2,
+    'no-lonely-if': 'error',
 
     // Force curly braces for control flow,
     // including if blocks with a single statement
-    curly: [2, 'all'],
+    curly: ['error', 'all'],
 
     // No async function without await
-    'require-await': 2,
+    'require-await': 'error',
 
     // Force dot notation when possible
-    'dot-notation': 2,
+    'dot-notation': 'error',
 
-    'no-var': 2,
+    'no-var': 'error',
 
     // Force object shorthand where possible
-    'object-shorthand': 2,
+    'object-shorthand': 'error',
 
     // No useless destructuring/importing/exporting renames
-    'no-useless-rename': 2,
+    'no-useless-rename': 'error',
+
+    /**********************/
+    /*   Unicorn Rules    */
+    /**********************/
+
+    // Pass error message when throwing errors
+    'unicorn/error-message': 'error',
+
+    // Uppercase regex escapes
+    'unicorn/escape-case': 'error',
+
+    // Use "new" for most builtins (not String, Number, ..., though)
+    'unicorn/new-for-builtins': 'error',
+
+    // Array.isArray instead of instanceof
+    'unicorn/no-array-instanceof': 'error',
+
+    // for-of instead traditional for loop
+    'unicorn/no-for-loop': 'error',
+
+    // Prevent deprecated `new Buffer()`
+    'unicorn/no-new-buffer': 'error',
+
+    // Keep regex literals safe!
+    'unicorn/no-unsafe-regex': 'off',
+
+    // Use non-fraction numbers if possible, e.g. 1 instead of 1.0
+    'unicorn/no-zero-fractions': 'error',
+
+    // Lowercase number formatting for octal, hex, binary (0x12 instead of 0X12)
+    'unicorn/number-literal-case': 'error',
+
+    // ** instead of Math.pow()
+    'unicorn/prefer-exponentiation-operator': 'error',
 
-    'vue/no-parsing-error': [2, {
+    // includes over indexOf when checking for existence
+    'unicorn/prefer-includes': 'error',
+
+    // append over appendChild
+    'unicorn/prefer-node-append': 'error',
+
+    // remove over removeChild
+    'unicorn/prefer-node-remove': 'error',
+
+    // querySelector instead of getElementBy...
+    'unicorn/prefer-query-selector': 'error',
```

Comment:
Do we really need to disallow a commonly used feature in flavor on other one?

Reply:
The rule is (partially) fixable and enforces consistency. I don't see a big reason against it. ☺️

Reply:
But if i use getEleementById no one can state it is a bad practice. A built in browser feature which is exactly doing what is expected to do!

Reply:
Never said it's a bad practice. But if there are two "equivalent" ways of doing things, why not stick to *one* of them?

Using both, though they aren't different, leads to more cognitive load (thinking about both ways and how they work instead of "just one").

Reply:
I wish they was equivalent but are not :) getElementById is a single selector.  It is like saying both if and switch case can do same thing. So disallow one.

Reply:
So is `querySelector` :yum: 
![image](https://user-images.githubusercontent.com/640208/56795929-d643a180-6811-11e9-9e8e-3868d0baa6b0.png)

However, I see your point here!

Reply:
MDN says that querySelector is a recommended method and documentByID is old one and  https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Client-side_web_APIs/Manipulating_documents#Active_learning_Basic_DOM_manipulation

Reply:
Also querySelector is a single selector too.

Reply:
Still getElementById is not discouraged by MDN or any other standard:
https://developer.mozilla.org/en-US/docs/Web/API/Document/getElementById

And also why should be included in nuxt preset at all ?!

Reply:
For consistency,  for example right now in nuxt codebase in one place we use getElementById, in others querySelector :)